nikola/data/themes/base/templates/index.tmpl

```diff
@@ -11,10 +11,24 @@
         <link rel="prefetch" href="${posts[0].permalink()}" type="text/html">
     % endif
     ${math.math_styles_ifposts(posts)}
+    %if other_languages is not UNDEFINED and other_languages:
```

Comment:
I've added `has_other_languages`. This means that now `list.tmpl` requires `has_other_languages` to be defined, which might be a problem for custom plugins. It looks like none of the standard plugins (both Nikola core and Nikola plugins repositories) do that except taxonomies.